scripts/build/build/targets.py

```diff
@@ -527,6 +528,13 @@ def Bl602Targets():
     yield target.Extend('light', board=Bl602Board.BL602BOARD, app=Bl602App.LIGHT)
 
 
+def IMXTargets():
+    target = Target('imx', IMXBuilder)
+
+    yield target.Extend('chip-tool', app=IMXApp.CHIP_TOOL)
+    yield target.Extend('lighting-app', app=IMXApp.LIGHT)
```

Comment:
Consider yiealding release variants as well since it seems the build configuration supports it:

```py
    yield target.Extend('chip-tool-release', app=IMXApp.CHIP_TOOL, release=True)
    yield target.Extend('lighting-app-release', app=IMXApp.LIGHT, release=True)
```

Reply:
done.